packages/bundler/src/test/es6-module-bundling-scenarios_test.ts

```diff
@@ -313,5 +312,38 @@ suite('Es6 Module Bundling', () => {
         };
         export { a$1 as $a, b$1 as $b, a, b };`);
     });
+
+    test('deduplicate static imports, not dynamic', async () => {
```

Comment:
would this test case catch both of the issues mentioned?

Reply:
This test covers the issue #568 which boils down to not letting rollup dedup dynamic imports.  The other issues addressed in the PR are tested in https://github.com/Polymer/tools/pull/722/files#diff-11afbe94c33f83818dca10824afa21b6R107 for example.
Other issues tackled are not surfaced easily in tests; they're more about changing the use of rollup under-the-hood to enable the other things.  I think the existing tests demonstrate there is no regression.

Is that sufficient?

Reply:
ok